engine_interface/src/lib.rs

```diff
@@ -67,3 +68,19 @@ impl Component for Saved {
         size: 0,
     };
 }
+
+/// Frame information
+#[derive(Serialize, Deserialize, Debug, Clone, Copy)]
+pub struct FrameTime {
```

Comment:
Maybe having a u64 of nanoseconds would be useful here in terms of precision jic someone wants to keep track of fine grained physics.

Reply:
Good point, I'll put a TODO on it

Reply:
🦀